### DISCUSSION TITLE
Add next/swc support for Android armv7

### DISCUSSION THREAD

**Discussion by melsiir** [vercel/next.js#33530] [Ideas]
### Describe the feature you'd like to request

I want next/swc support for Android armv7 to make swc compiler available on Android armv7

### Describe the solution you'd like

There already swc binary for Android armv7 and I use it with typescript : 
https://www.npmjs.com/package/@swc/core-android-arm-eabi

But there are no  next/swc binary for it  I think there are  should be next/swc binary For Android armv7 so I can use swc compiler with next instead of babel

### Describe alternatives you've considered

I using babel for now

**Comment by melsiir** [comment]
Waiting ....

**Comment by Brooooooklyn** [comment]
Fixed in https://github.com/vercel/next.js/pull/34481
